### ISSUE TITLE
Parsing Issue

### ISSUE THREAD

**Issue by leondexelle** [yargs/yargs-parser#138]
The parser seems to remove quotes before passing in any specific options.
My use case is creating a sort of evaluation command which parses in certain options from the user like so:
User: `someExampleFunction('hello')`
But, I get this:
Error: `ReferenceError: hello is not defined`

The actual parsed string is `someExampleFunction(hello)`.
Is there any way to get around this and stop it removing quotes before I add something like `--some-option "testing"` (Where I would want it to set some-option to "testing")?

**Comment by bcoe** [comment]
@leondexelle I apologize that it's taken me so long to sift through old issues; could you try the latest version of yargs-parser and let me know if this is still happening to you? we've made a few fixes around tokenization in recent months.

**Comment by leondexelle** [comment]
I tried this again with v11.1.1 and I'm getting the exact same output
```js
require('yargs-parser')('someFunction("hello")')
```
returns:
```js
{ _: [ 'someFunction(hello)' ] }
```

and I want it to be
```js
{ _: [ 'someFunction("hello")' ] }
```

**Comment by bcoe** [comment]
@leondexelle this should now be addressed in `+ yargs-parser@12.0.0`, please take it for a spin 👍 

```sh
> require('yargs-parser')('someFunction("hello")')
{ _: [ 'someFunction("hello")' ] }
```

let me know if you continue to bump into any issues.
